### PR TITLE
Feature/#7 normalization

### LINKED ISSUE #7: Don't remove character during normalization
I would like to my upgrade my projects to the latest version but this introduce a blocker for me. IMHO `:class` is totally different from `class` and this should not be normalized.

```
info = find(schema, ':class')
info.attribute // className
```

Current
```
normalize('d-class') => dclass
normalize(':class') => class
normalize('CLASS') => class
```

I suggest to update the normalize function to because this reflects the current spec.

```
function normalize(value) {
  return value.toLowerCase()
}
```

After
```
normalize('d-class') => d-class
normalize(':class') => :class
normalize('CLASS') => class
```

### PR DESCRIPTION
**Pull request by StarpTech**
* Fix #7 
* Fix scripts args in order to support windows systems

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
   },
   "scripts": {
     "generate": "node script/generate-react",
-    "format": "remark . -qfo && prettier --write '**/*.js' && xo --fix",
+    "format": "remark . -qfo && prettier --write \"**/*.js\" && xo --fix",
```
Comment (wooorm) [inline package.json:56]: Do single quotes not work? 🤔

File: test.js [hunk 2]
```diff
@@ -184,6 +184,27 @@ test('normalize', function(t) {
     'properties should match attributes (#5)'
   )
 
+  t.equal(
+    normalize(':class'),
+    ':class',
+    `non-word characters should not be removed
+    when they did not serve as a delimiter (#7)`
```
Comment (wooorm) [inline test.js]: Could you use the format `(GH-7, #1)` here and the two below? I use the `#` above as a count.

File: package.json (hunk 1 above)
Comment (StarpTech) [reply package.json:56]: No, it's a common behavior. 

* https://github.com/keithamus/npm-scripts-example/issues/5
* https://github.com/Jephuff/fix-windows-single-quotes#readme

**Comment by StarpTech** [comment]
Done.

**Comment by StarpTech** [comment]
Why did you close it?

**Comment by wooorm** [comment]
because I merged it

**Comment by StarpTech** [comment]
ok, for me it's highlighted as unmerged and closed.

**Comment by wooorm** [comment]
Yes, but GH UI is lacking so I merge stuff often on the terminal.

**Pull request closed without merging**